docs/tutorial.md

```diff
@@ -97,10 +96,9 @@ corporate website with a home page, an About page, different sections, and a blo
 
 ## Download Gutenberg
 
-The first thing to do is obtain the Gutenberg static site generator. It is small and fast, and because it's a command-line utility written in Rust, Gutenberg works on all operating systems that have a Rust compiler and a web server available to them. You don't need to know Rust to use Gutenberg effectively, although understanding Rust's templating will make it easier
-for you to create Gutenberg themes.
+The first thing to do is obtain the Gutenberg static site generator binary. It is small and fast, and because it's a command-line utility written in Rust, Gutenberg works on all operating systems that Rust supports.
 
-At the time of writing, Gutenberg lacks installers for most operating systems but that barely matters. It consists of a single executable file you run from the command line.
```

Comment:
As you said before, Gutenberg is more of programmer tool and programmers are more likely to use package managers than installers I think.